README.md

```diff
@@ -0,0 +1,68 @@
+# Apple Income Statements
+
+This is a project developed in React and Rudux, that uses the [Financial Modeling Prep API](https://site.financialmodelingprep.com/developer/docs) to create a single page application that display each statement of the last 5 years. The design was inspired by Nelson Sakwa's [design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).
+
+
+## Screenshot
+
+![screenshot](./public/apple.png)![screenshot](./public/apple1.png)
+
+
+## Built With
+
+- HTML 
+- CSS 
+- JavaScript
+- React
+- Redux
+
+## Live Demo
+
+[GitHub Link](https://fabianofrank.github.io/apple-income/)
+
+
+## Getting Started
+
+To get a local copy up and running follow the instructions.
+
+
+### Setup
+
+1. Open the terminal and clone the project using `git clone git@github.com:fabianofrank/apple-income.git`
+
+
+### Install
+
+1. `cd` into the project folder and run `npm install`
+2. Run the command `npm start`
+
+
+### Run tests
+
+1. Run the command `npm test`
+
+
+## Author
+
+👤 **Fabiano Frank**
+
+- GitHub: [@fabianofrank](https://github.com/fabianofrank)
+- Twitter: [@fabianofrankk](https://twitter.com/fabianofrankk)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/fabianofrank/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
```

Comment:
- [ ] As part of the project requirements, pease add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).